src/Analyzer.Core.JsonRuleTests/Analyzer.Core.JsonRuleTests.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.9.4" />
+    <PackageReference Include="MSTest.TestAdapter" Version="2.2.3" />
+    <PackageReference Include="MSTest.TestFramework" Version="2.2.3" />
+    <PackageReference Include="coverlet.collector" Version="3.0.2" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Analyzer.Core\Analyzer.Core.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Update="Tests\*">
```

Comment:
Maybe we want a folder per built-in rule, to store the test templates and the file with the test information? Because we would like minimal test templates, right? So there shouldn't be that much template reuse

Reply:
I organized them this way because I wanted to avoid a bunch of nested directories - it looks cleaner to me being just one directory.

The effort involved in creating test templates isn't something I have a clear picture on yet.  Ideally as little effort as possible.  Does that mean many simple templates?  Or fewer templates that define everything tests need?  Probably a combination of both.  I don't want to prevent tests from taking advantage of test templates that already exist that provide everything needed.

Template naming also needs to be considered.  When multiple templates contain almost identical contents, but differ slightly for test purposes, how can those be differentiated in a meaningful way without having almost sentence-long names?  This was one reason I considered the desire to combine some scenarios into single templates.

Reply:
mmm I can see how having test templates ready could simplify writing some test cases, but for edge cases I imagine we would end up creating specific long name templates anyways